osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/misc/GetPowerQualityProfileCommandExecutor.java

```diff
@@ -25,28 +25,31 @@
 import org.opensmartgridplatform.adapter.protocol.dlms.domain.commands.AbstractCommandExecutor;
 import org.opensmartgridplatform.adapter.protocol.dlms.domain.commands.utils.JdlmsObjectToStringUtil;
 import org.opensmartgridplatform.adapter.protocol.dlms.domain.entities.DlmsDevice;
+import org.opensmartgridplatform.adapter.protocol.dlms.domain.entities.Protocol;
 import org.opensmartgridplatform.adapter.protocol.dlms.domain.factories.DlmsConnectionManager;
 import org.opensmartgridplatform.adapter.protocol.dlms.exceptions.ConnectionException;
 import org.opensmartgridplatform.adapter.protocol.dlms.exceptions.ProtocolAdapterException;
 import org.opensmartgridplatform.dto.valueobjects.smartmetering.ActionResponseDto;
 import org.opensmartgridplatform.dto.valueobjects.smartmetering.CaptureObjectDefinitionDto;
-import org.opensmartgridplatform.dto.valueobjects.smartmetering.DefinableLoadProfileConfigurationDto;
+import org.opensmartgridplatform.dto.valueobjects.smartmetering.GetPowerQualityProfileRequestDto;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
 @Component
-public class ConfigureDefinableLoadProfileCommandExecutor
-        extends AbstractCommandExecutor<DefinableLoadProfileConfigurationDto, Void> {
+public class GetPowerQualityProfileCommandExecutor
+        extends AbstractCommandExecutor<GetPowerQualityProfileRequestDto, Void> {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(GetPowerQualityProfileCommandExecutor.class);
 
     private static final int CLASS_ID = InterfaceClass.PROFILE_GENERIC.id();
-    private static final ObisCode LOGICAL_NAME = new ObisCode("0.1.94.31.6.255");
 
-    private static final AttributeAddress ATTRIBUTE_CAPTURE_OBJECTS = new AttributeAddress(CLASS_ID, LOGICAL_NAME,
-            ProfileGenericAttribute.CAPTURE_OBJECTS.attributeId());
-    private static final String ATTRIBUTE_NAME_CAPTURE_OBJECTS = "capture objects";
+    private static final ObisCode OBIS_CODE_DEFINABLE_LOAD_PROFILE = new ObisCode("0.1.94.31.6.255");
+    private static final ObisCode OBIS_CODE_PROFILE_1 = new ObisCode("1.0.99.1.1.255");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "OBIS_CODE_PROFILE_1" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1068)